lib/vagrant-bhyve/driver.rb

```diff
@@ -225,29 +228,16 @@ def enable_nat(bridge, ui)
 	execute(false, "#{@sudo} sysctl net.inet.ip.forwarding=1 >/dev/null 2>&1")
 	execute(false, "#{@sudo} sysctl net.inet6.ip6.forwarding=1 >/dev/null 2>&1")
 
-	# Change pf's configuration
-	pf_conf = directory.join("pf.conf")
-	pf_conf.open("w") do |pf_file|
-	  pf_file.puts "set skip on #{bridge_name}"
+        check_or_create_default_pfconf(ui)
+	# set up bridge pf anchor
+        pf_bridge_conf = "/usr/local/etc/pf.#{bridge_name}.conf"
```

Comment:
Arguably this should also be somewhere in `@data_dir`, as it's not really permanent configuration...